frontend/src/lib/components/DefinitionPopover/DefinitionPopoverContents.tsx

```diff
@@ -174,19 +174,23 @@ function DefinitionView({ group }: { group: TaxonomicFilterGroup }): JSX.Element
             </>
         )
     }
-    if (isProperty) {
+
+    const hasSentAsLabel = useMemo(() => {
```

Comment:
I think this still will bork because of the early returns above?

Reply:
Oh poop, I didn't see those, one sec..

Reply:
@neilkakkar sorted